addons/html_editor/static/src/core/selection_plugin.js

```diff
@@ -489,14 +517,14 @@ export class SelectionPlugin extends Plugin {
      * @param { number } [selection.focusOffset=selection.anchorOffset]
      * @param { Object } [options]
      * @param { boolean } [options.normalize=true] Normalize deep the selection
-     * @return { EditorSelection }
+     * @return { EditorSelection | null }
      */
     setSelection(
         { anchorNode, anchorOffset, focusNode = anchorNode, focusOffset = anchorOffset },
         { normalize = true } = {}
     ) {
-        if (!this.isSelectionInEditable({ anchorNode, focusNode })) {
-            throw new Error("Selection is not in editor");
+        if (!this.validateSelection({ anchorNode, anchorOffset, focusNode, focusOffset })) {
+            return null;
```

Comment:
Why explicitely return `null` instead of the implicit `undefined` from a simple `return`?